website/docs/build-apps/how-to-guides/import-export-app-entities.md

```diff
@@ -0,0 +1,33 @@
+---
+description: A guide that outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+---
+
+# Reuse App Entities
+
+When working on complex Appsmith applications, you might find the need to reuse a particular set of widgets, queries, JSObjects, and more across pages. Instead of recreating these entities from scratch each time, Appsmith provides a way for you to export entities from one page and import them into another. This guide outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+
+## Export entities
```

Comment:
There is no need for subsections in this guide because it is a complete guide. It's not an option. As in they have to export followed by import. Also because users don't understand that we facilitate reuse by import and export, giving those as headings don't make sense.